provider/rfc2136/rfc2136.go

```diff
@@ -439,3 +445,18 @@ func chunkBy(slice []*endpoint.Endpoint, chunkSize int) [][]*endpoint.Endpoint {
 
 	return chunks
 }
+
+func findMsgZone(ep *endpoint.Endpoint, m *dns.Msg, r *rfc2136Provider) {
+	for _, zone := range r.zoneNames {
+		if strings.HasSuffix(ep.DNSName, zone) {
+			r.krb5Realm = strings.ToUpper(dns.Fqdn(zone))
+			m.SetUpdate(dns.Fqdn(zone))
+
+			return
+		}
+	}
+
+	log.Debugf("No available zone found for %s, set it to 'root'", ep.DNSName)
```

Comment:
Shouldn't this be a warning ?

Reply:
I think yes. It's a debug message.

Reply:
fixed https://github.com/kubernetes-sigs/external-dns/pull/3976/commits/714078dc95db9514e4613502f42f7efe7c0db10e